eZ/Publish/Core/FieldType/DateAndTime/Type.php

```diff
@@ -148,7 +148,10 @@ protected function getSortInfo(BaseValue $value)
     /**
      * Converts an $hash to the Value defined by the field type.
      *
-     * @param mixed $hash Null or associative array containing timestamp and optionally date in RFC850 format.
+     * @param mixed $hash Null or associative array containing timestamp and optionally date in RFC850 format,
+     *                    and optionally date in parseable string format (e.g. 'now', '+3 days')
+     *
+     * @deprecated timestamp is no longer used and will be removed in a future version.
```

Comment:
in correct here, it is still in use, just not if any of the other are defined.

So: comment is correct below, and that is where I had in mind it being deprecated. Comment here needs to be adjusted, or maybe even removed in favor of describing which order the options are taken _(keeping timestamp here is not a problem, as long as the others are picked first)_